lms/djangoapps/course_home_api/progress/v1/views.py

```diff
@@ -109,12 +125,26 @@ def get(self, request, *args, **kwargs):
         course_grade = CourseGradeFactory().read(request.user, course)
         courseware_summary = course_grade.chapter_grades.values()
 
+        verification_status = IDVerificationService.user_status(request.user)
+        verification_link = None
+        if verification_status['status'] is None or verification_status['status'] == 'expired':
+            verification_link = IDVerificationService.get_verify_location('verify_student_verify_now', course_id=course_key)
+        elif verification_status['status'] == 'must_reverify':
+            verification_link = IDVerificationService.get_verify_location('verify_student_reverify', course_id=course_key)
+        verification_data = {
+            'link': verification_link,
+            'status': verification_status['status'],
+            'status_date': verification_status['status_date'],
+        }
```

Comment:
Nit: Put a new line between this and data just to keep everything in its own chunk.